### ISSUE TITLE
Training Transfer Learning

### ISSUE THREAD

**Issue by nurul274** [fadilahnurkh09/intprog2-project#20]
Melatih MobileNetV2 menggunakan transfer learning agar model dapat mempelajari karakteristik lima kelas bentuk wajah.

- [ ] Memuat pretrained MobileNetV2
- [ ]  Membekukan layer awal model
- [ ]  Menambahkan classification head
- [ ]  Menentukan lima output class
- [ ]  Menerapkan softmax
- [ ]  Melakukan proses training
- [ ]  Menyimpan hasil training
- [ ]  Mencatat training history
